player/lua/console.lua

```diff
@@ -691,28 +691,28 @@ function build_completers()
         }
     end
 
-    for _, command in pairs({'set', 'cycle[-_]values'}) do
+    for _, command in pairs({'set'}) do
         completers[#completers + 1] = {
             pattern = '^%s*' .. command .. '%s+"?([%w_-]+)"?%s+"()%S*$',
             list = choice_list,
-            append = command == 'cycle[-_]values' and '" ' or '"',
+            append = '"',
         }
         completers[#completers + 1] = {
             pattern = '^%s*' .. command .. '%s+"?([%w_-]+)"?%s+()%S*$',
             list = choice_list,
-            append = command == 'cycle[-_]values' and ' ' or nil,
+            append = nil,
         }
     end
 
     completers[#completers + 1] = {
-        pattern = '^%s*cycle[-_]values%s+"?([%w_-]+)"?%s+%S+%s+"()%S*$',
+        pattern = '^%s*cycle[-_]values%s+"?([%w_-]+)"?[%s%S]-%s+"()%S*$',
```

Comment:
Just use `.-`?

Reply:
omg I can't believe I wrote it like that :rofl:
That comes from making quick PRs while tired.